docs/bucket/notification/README.md

```diff
@@ -0,0 +1,113 @@
+# Minio Event Notification
+
+Minio object storage server supports [AWS S3 SQS/SNS style](http://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html) event notification for object create and delete operations. Notifications can be received through AMQP, Elasticsearch and Redis targets.
+
+## Supported Events
+
+Currently, Minio can publish the following events:
+
+- _ObjectCreated_ event is generated whenever a new object is created.
+
+| Event types| Description|
+|:-----------|:-----------|
+|_s3:ObjectCreated:*_| All object create operations.|
+|_s3:ObjectCreated:Put_| Object created via PUT method. (Most commonly used by SDKs)|
+|_s3:ObjectCreated:Post_| Object created via POST method. (Most commonly used by browser)|
+|_s3:ObjectCreated:Copy_| Object created via server side copy operation.|
+|_s3:ObjectCreated:CompleteMultipartUpload_| Object created via Multipart API. (Most commonly used for large objects)|
+
+- _ObjectRemoved_ event is generated whenever an object is deleted.
+
+| Event types| Description|
+|:-----------|:-----------|
+|_s3:ObjectRemoved:*_| All delete operations.|
+|_s3:ObjectRemoved:Delete_||
+
+## Supported Targets
+
+Minio can send bucket notifications to one or more of the following supported targets. The target is identified uniquely through an unique resource identifier called [ARN](http://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html).
```

Comment:
We are switching between event notifications and bucket notifications. We should either use one of them everywhere or explain how they could be used interchangeably.